exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return number > 0 && number < 10;
+}
```

Comment:
Do you want the range to include 0 and 10? The question doesn't make it clear, so it's up to you, but if you want 0 and 10 to return true, you'll need to change > and <